hazelcast/src/main/java/com/hazelcast/core/IMap.java

```diff
@@ -346,13 +500,19 @@
     void loadAll(Set<K> keys, boolean replaceExistingValues);
 
     /**
-     * Clears the map and invokes {@link MapStore#deleteAll} which,
-     * if connected to a database, will delete the records from the database.
+     * Clears the map and deletes the items from the backing map store.
      * <p>
      * The MAP_CLEARED event is fired for any registered listeners.
      * See {@link com.hazelcast.core.EntryListener#mapCleared(MapEvent)}.
      * <p>
-     * To clear a map without calling {@link MapStore#deleteAll}, use {@link #evictAll}.
+     * To clear a map without removing the items from the map store too,
```

Comment:
"To clear a map" -> "To clear the map"
remove "too", or use also: "without also removing..."